src/main/java/org/jarvis4ops/controller/SlachController.java

```diff
@@ -205,5 +207,35 @@ public String postMaxWipBreachedSlack(@RequestBody String jiraMaxWipCountRequest
 		slackHelper.postOnSlack(gson.toJson(slachAttachments), "jiraBots");
 		return "200";
 	}
+	
+	@RequestMapping(value = "/postInvalidTimeOnSlack", method = { RequestMethod.POST })
+	public String slackMessageForInvalidTime(@RequestBody String invalidJiraTimeIssues) {
+		
+		Gson gson = new Gson();
+		Type newType = new TypeToken<HashMap<String, TimeBean>>(){}.getType();
+		HashMap<String, TimeBean> map = new Gson().fromJson(invalidJiraTimeIssues, newType);
+		
+		slackHelper.getInvlaidTimes(map);
```

Comment:
Spelling - and also invalidTimes is not a correct variable we should use - as there is no invalid timing